internal/livechat/sdk/client.go

```diff
@@ -0,0 +1,115 @@
+package sdk
```

Comment:
why not just use `github.com/livechat/lc-sdk-go/v6`

Reply:
Since I had already used the SDK in the PoC for testing in golang. This time I wanted to try working directly with the HTTP client, creating requests and passing them to .Do(). Including sending the request body and decoding the response into a structure containing only the fields I actually need etc. I borrowed some good practices from the SDK regarding defining RequestPayload/Response but it still was valuable for me compared to just relying on the ready for use SDK.